drivers/storage/portworx/util/util.go

```diff
@@ -132,6 +132,8 @@ const (
 	AnnotationClusterID = pxAnnotationPrefix + "/cluster-id"
 	// AnnotationPreflightCheck do preflight check before installing Portworx
 	AnnotationPreflightCheck = pxAnnotationPrefix + "/preflight-check"
+	// AnnotationIsAutoTLS annotation whether to set up cluster with automatic SSL/TLS credentials
+	AnnotationIsAutoTLS = pxAnnotationPrefix + "/is-auto-tls"
```

Comment:
nit: we have is-eks, is-iks flags, I feel probably it's better to simply use `auto-tls` instead of `is-auto-tls`? it's a feature to enable or disable.